plugins/techdocs-node/src/stages/publish/awsS3.test.ts

```diff
@@ -478,7 +478,7 @@ describe('AwsS3Publish', () => {
 
       await expect(fails).rejects.toMatchObject({
         message: expect.stringMatching(
-          /TechDocs metadata fetch failed; caused by Error: The file .* does not exist/i,
```

Comment:
This looks a little strange that we're closing the scope for the regex when we're adding the ability to log more info. Is there a way that we can include the error message in this regex?

Reply:
Sure, the error is from the aws-sdk getObject mockup in [L64](https://github.com/nokia/backstage/blob/6c118dd093eea6f96004bb8ee3a48a8391dcdb8d/plugins/techdocs-node/src/stages/publish/awsS3.test.ts#L62-L65), so we can pin it to the exact error message.

Reply:
no regex needed